cmake/Modules/FindAD9361.cmake

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 Free Software Foundation, Inc.
```

Comment:
@adamhorden  if you wrote those, please change them to 
```suggestion
# Copyright 2021 Adam Horden
```

If you took them over this way, then just leave them and ignore this comment.

Reply:
Thanks for spotting this 🤓 . 
I wrote these CMake build files to ensure a required dependancy on libiio and optional dependencies on libad9361.
Updated in a separate commit that can be rebased 😄 .